tests/tensorflow/quantization/test_builder_state.py

```diff
@@ -148,179 +136,6 @@ def test_checkpoint_callback_make_checkpoints(mocker, tmp_path):
-                "instance_index": 0,
-                "layer_name": "dummy_str"
-            },
-            "target_point_class_name": "TFBeforeLayer"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "instance_index": 0,
-                "layer_name": "dummy_str",
-                "out_port": 0
-            },
-            "target_point_class_name": "TFAfterLayer"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "layer_name": "dummy_str",
-                "weights_attr_name": "dummy_str"
-            },
-            "target_point_class_name": "TFLayerWeight"
-        },
-        {
-            "op_name": "dummy_str",
-            "quantizer_spec": {
-                "half_range": True,
-                "mode": "symmetric",
-                "narrow_range": True,
-                "num_bits": 8,
-                "per_channel": True,
-                "signedness_to_force": None
-            },
-            "target_point": {
-                "layer_name": "dummy_str",
-                "operation_name": "dummy_str",
-                "weights_attr_name": "dummy_str"
-            },
-            "target_point_class_name": "TFOperationWithWeights"
-        }
-    ]
-}
-
-
-def test_quantizer_setup_serialization():
-    setup = QuantizationSetup()
-
-    quantizer_spec = TFQuantizerSpec(num_bits=8, mode=QuantizationMode.SYMMETRIC, signedness_to_force=None,
-                                     narrow_range=True, half_range=True, per_channel=True)
-    check_serialization(quantizer_spec)
-
-    target_type = TargetType.OPERATOR_POST_HOOK
-    check_serialization(target_type)
-
-    quantization_point_factory = partial(QuantizationPoint, op_name=DUMMY_STR, quantizer_spec=quantizer_spec)
-    check_insertion_point = partial(_check_and_add_insertion_point, quantization_point_factory, setup)
-
-    check_insertion_point(TargetPoint(target_type))
-    check_insertion_point(TFLayerPoint(target_type, layer_name=DUMMY_STR))
-    check_insertion_point(TFLayer(layer_name=DUMMY_STR))
-    check_insertion_point(TFBeforeLayer(layer_name=DUMMY_STR))
-    check_insertion_point(TFAfterLayer(layer_name=DUMMY_STR))
-    check_insertion_point(TFLayerWeight(layer_name=DUMMY_STR, weights_attr_name=DUMMY_STR))
-    check_insertion_point(TFOperationWithWeights(layer_name=DUMMY_STR, weights_attr_name=DUMMY_STR,
```

Comment:
BTW, is there a similar test for all possible insertion points in TF?

Reply:
Restored